dojo/tools/nmap/parser.py

```diff
@@ -46,8 +46,15 @@ def get_findings(self, file, test):
             if fqdn is not None:
                 host_info += f"**FQDN:** {fqdn}\n"
 
-            host_info += "\n\n"
+            url = (
+                host.find("hostnames/hostname[@type='user']").attrib["name"]
+                if host.find("hostnames/hostname[@type='user']") is not None
+                else None
+            )
```

Comment:
The existing code has hostnames of type PTR. This PR adds type USER. But what if there are more types now or in the future? Shouldn't the parser just add all hostnames regardless of type?

Reply:
This is a good idea if it is feasible